cfme/fixtures/vm.py

```diff
@@ -11,7 +12,15 @@
 
 def _create_vm(request, template, provider, vm_name):
     collection = provider.appliance.provider_based_collection(provider)
-    vm_obj = collection.instantiate(vm_name, provider, template_name=template.name)
+    try:
+        assignee = DocstringParser(request.function.__doc__).parse()["assignee"].value
+    except TypeError:
+        assignee = ""
+    if assignee:
+        vm_name = vm_name + "-" + assignee
```

Comment:
I think the logic can be simplified a bit here to avoid adding these strings in a clunky way, and do it more consistently.

Consider using `'unassigned'` or something similar when its missing.